euth/dashboard/forms.py

```diff
@@ -109,10 +116,115 @@ class Meta:
         fields = ['name', 'description', 'image', 'information', 'is_public',
                   'result']
 
+    def send_to_flashpoll(self):
```

Comment:
This function is a bit long. Maybe all the request sending could be factored out into a `FlashpollClient` class.